internal/version/version_test.go

```diff
@@ -12,6 +12,62 @@ var _ = t.Describe("Version", func() {
 	tempFileName := "tempVersionFile"
 	tempVersion := "1.1.1"
 	tempVersion2 := "1.13.1"
+	testInfo := Info{
+		Version:         "1.0.0",
+		GitCommit:       "abcdef123456",
+		GitCommitDate:   "2024-08-13T12:34:56Z",
+		GitTreeState:    "clean",
+		BuildDate:       "2024-08-13T12:34:56Z",
+		GoVersion:       "go1.20.5",
+		Compiler:        "gc",
+		Platform:        "linux/amd64",
+		Linkmode:        "external",
+		BuildTags:       []string{"tag1", "tag2"},
+		LDFlags:         "-X main.Version=1.0.0",
+		SeccompEnabled:  true,
+		AppArmorEnabled: false,
+		Dependencies:    []string{"dep1", "dep2"},
+	}
+	testInfoStr := "Version:        1.0.0\n" +
+		"GitCommit:      abcdef123456\n" +
+		"GitCommitDate:  2024-08-13T12:34:56Z\n" +
+		"GitTreeState:   clean\n" +
+		"BuildDate:      2024-08-13T12:34:56Z\n" +
+		"GoVersion:      go1.20.5\n" +
+		"Compiler:       gc\n" +
+		"Platform:       linux/amd64\n" +
+		"Linkmode:       external\n" +
+		"BuildTags:      \n" +
+		"  tag1\n" +
+		"  tag2\n" +
+		"LDFlags:          -X main.Version=1.0.0\n" +
+		"SeccompEnabled:   true\n" +
+		"AppArmorEnabled:  false\n" +
+		"Dependencies:     \n" +
+		"  dep1\n" +
+		"  dep2"
+	testJSONInfoStr := "{" + "\n" +
+		"  \"version\": \"1.0.0\"," + "\n" +
+		"  \"gitCommit\": \"abcdef123456\"," + "\n" +
+		"  \"gitCommitDate\": \"2024-08-13T12:34:56Z\"," + "\n" +
+		"  \"gitTreeState\": \"clean\"," + "\n" +
+		"  \"buildDate\": \"2024-08-13T12:34:56Z\"," + "\n" +
+		"  \"goVersion\": \"go1.20.5\"," + "\n" +
+		"  \"compiler\": \"gc\"," + "\n" +
+		"  \"platform\": \"linux/amd64\"," + "\n" +
+		"  \"linkmode\": \"external\"," + "\n" +
+		"  \"buildTags\": [" + "\n" +
+		"    \"tag1\"," + "\n" +
+		"    \"tag2\"" + "\n" +
+		"  ]," + "\n" +
+		"  \"ldFlags\": \"-X main.Version=1.0.0\"," + "\n" +
+		"  \"seccompEnabled\": true," + "\n" +
+		"  \"appArmorEnabled\": false," + "\n" +
+		"  \"dependencies\": [" + "\n" +
+		"    \"dep1\"," + "\n" +
+		"    \"dep2\"" + "\n" +
+		"  ]" + "\n" +
+		"}"
```

Comment:
This is not how Go developers would often approach larger blocks of text.

Reply:
I thought it just looked better now using text block.